docs/src/transforms/builtin.md

```diff
@@ -163,3 +163,15 @@ RowTable
 ```@docs
 ColTable
 ```
+
+## SequentialTransform
+
+```@docs
+TransformsBase.SequentialTransform
+```
+
+## ParallelTableTransform
+
+```@docs
+TableTransforms.ParallelTableTransform
+```
```

Comment:
```suggestion
```

These transforms are internal implementation detail, they are not intended for end users who are invited to use the \to and \sqcup instead.